utils/log.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+func GetLogFileName(t *testing.T, label string) string {
+	fileName := strings.ReplaceAll(t.Name(), "/", "-") + "-" + label + ".log"
+	logDirectory := "logs"
+
+	err := os.MkdirAll(logDirectory, 0777)
+	if err != nil {
+		t.Fatalf("can't create log directory")
```

Comment:
Error logs lines are standalone sentences.
```suggestion
		t.Fatalf("Can't create log directory")
```

Reply:
I didn't know this. I was following the error message style. But this is well documented [here](https://google.github.io/styleguide/go/decisions.html#:~:text=t.Errorf(%22Op(%25q)%20failed%20unexpectedly%3B%20err%3D%25v%22%2C%20args%2C%20err)), so I will change:
> On the other hand, the style for the full displayed message (logging, test failure, API response, or other UI) depends, but should typically be capitalized.